src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -790,4 +790,13 @@ export class BrowsingContextImpl {
       },
     };
   }
+
+  async close(): Promise<void> {
+    if (!this.isTopLevelContext()) {
+      throw new Message.InvalidArgumentException(
+        'A top-level browsing context cannot be closed.'
```

Comment:
```suggestion
        'Non top-level browsing context ${context.id} cannot be closed.'
```

Reply:
This is resolved but it wasn't push. I just want to make sure it's addressed

Reply:
Ops forced pushed over it.